test/common/wpt.js

```diff
@@ -269,8 +269,16 @@ class StatusLoader {
 
   load() {
     const dir = path.join(__dirname, '..', 'wpt');
-    const statusFile = path.join(dir, 'status', `${this.path}.json`);
-    const result = JSON.parse(fs.readFileSync(statusFile, 'utf8'));
+    const statusFile = path.join(dir, 'status', `${this.path}.js`);
+    let result;
+    try {
+      result = require(statusFile);
+    } catch (err) {
+      if (err.code !== 'MODULE_NOT_FOUND') {
```

Comment:
Why is there a fallback now? I think it's better to continue to throw if the file doesn't exist (we can use `fs.statSync` to check for it too).

Reply:
Isn't status file presence optional?

Reply:
done.